Personalizing your Skywallet.md

```diff
@@ -0,0 +1,110 @@
+
+To create a PIN for your Skywallet for the first time, click on the button - **"Create PIN code"**
+
+Once you click on the button the following window will appear on the desktop wallet:
+
+<Screenshot showing the window to create a PIN> 
+<it is mentioned in the wiki page that the hash symbol which is currently used to mask the numbers in the keypad will be changed to dot, if so then the latest version screenshot will be needed>
+
+Simultaneously a number **Matrix** will be displayed on your Skywallet display screen.
+
+<Picture of the Skywallet showing the number matrix>
+
+In order to set the PIN, you have to use both the matrix as well as the desktop wallet window.
+
+**Each position of the number in the matrix corresponds to its position in the window shown by the Desktop wallet.**
+
+<A side by side picture of the two pictures mentioned above>
+
+<An explanation of this side by side picture>
+<The explanation can also be an edited image showing the correct position of the numbers in the masked keypad of the desktop wallet window>
+
+Alternatively, you can utilize the number keypad on your computer for entering the PIN. 
+
+However, if you are using the number pad then note that **the position of the number in the Matrix is what decides which number you are entering.**
+
+<Reusing the picture of the matrix>
+
+<An example comment using a single number on the position of the number keypad and the position of number in the matrix>
+
+***Note - While setting up your PIN you have to enter it twice.***
+
+After setting up your PIN you can manage the PIN by changing or deleting the same. You can read more about it in this section (Managing your PIN)[https://github.com/skycoin/hardware-wallet/wiki/Manage-the-PIN-code]
+
+Also to ensure optimal security for your funds, please make sure to go through this section of the user manual - (Optimize account security)[https://github.com/skycoin/hardware-wallet/wiki/Optimize-account-security]
+
+## Wipe the device
+
+**"Wipe the device"** button helps you to completely delete all the data on your Skywallet.
+
+Once you click on this button you will be asked for a confirmation message on both the Skywallet as well as Desktop Wallet.
+
+While confirming, note that there is an option to remove the Skywallet from your wallet list in the desktop wallet, by default this option will be checked and if not unchecked your Skywallet will be removed from the wallet list.
+
+After confirming the same, your Skywallet will be wiped off of all the data and you can use it as a brand new one.
+
+You can also restore all your coins by using the previously setup seed and the restore backup option, details on how to restore your wallet are available on this section (Restore Backup)[https://github.com/skycoin/hardware-wallet/wiki/Restore-configuration].
+
+**Note - Wiping the Skywallet will not delete the firmware updates that have been applied to the device.**
```

Comment:
Then that  means that not all the data on the device has been deleted. Please modify then, what data is deleted in the process.